### PR TITLE
Merging Savannah branch into webOps

### PR DESCRIPTION
**Pull request by SavannahRicks**
Reworking the menu bar!

### CODE REVIEW

File: docs/about.html [hunk 1]
```diff
@@ -23,10 +23,11 @@
...
             <div class="menu" id="myMenu">
                 <a href="index.html">Home</a>
+                <a href="showcase.html">Our Students</a>
+                <a href="roadmap.html">Education</a>
                 <a href="about.html" class="active">About</a>
-                <a href="roadmap.html">Courses</a>
-                <a href="showcase.html">Students Projects</a>
-                <a href="ev.html">Events</a>
+                <!-- <a href="ev.html">Events</a> -->
+                <a href="https://newtfire.org/courses/DIGITShowcase.html" target="_blank">DIGIT Works</a>
```
Comment (ebeshero) [inline docs/about.html:30]: @SavannahRicks I'm not sure this is the best idea to have the last menu bar item jump straight to my boring newtfire schedule for the current DIGIT Works... Maybe:
* Just keep it as an Events page (b/c we might conceivably want to advertise other events besides DIgit Works)
* Wasn't @rcc5418 going to just put an iframe on the Events page that would provide a view of the latest DIGIT Works schedule, and a means to navigate there?

**Review by ebeshero — CHANGES_REQUESTED** [review]

**Comment by ebeshero** [comment]
Okay, I found the ev.html page in docs, and it works okay. Suggestions if we want to keep it (put the link back in the menu bar):
* I know there are other events going on besides DIGIT works, such as a planned international film festival this fall
* We could use this page to provide suggestions, like once you're here you can the DIGIT Canvas pride group
* Having an ev.html page makes it possible to add even simple links to it, even if it's a little boring for now.

File: docs/about.html (hunk 1 above)
Comment (SavannahRicks) [reply docs/about.html]: Yeah, we can keep the Events page, I can have the link to your Newtfire <a> tagged in at the bottom of the page

**Review by SavannahRicks — COMMENTED** [review]
I just removed the commented Events tab in my latest push!

**Comment by SavannahRicks** [comment]
The merge conflict is an easy one

File: docs/about.html (hunk 1 above)
Comment (ebeshero) [reply docs/about.html]: I just pulled in the changes and it all looks good! Thanks @SavannahRicks!

**Review by ebeshero — APPROVED** [review]
Thanks for updating the menu link to the ev.html page!

**Pull request merged**